LICENSE-CODE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 GitHub Community
```

Comment:
```suggestion
Copyright (c) GitHub
```

Reply:
I'm the onwer of accounts and y'all help someone steal from me